api/src/test/java/com/epam/pipeline/controller/entity/EntityControllerTest.java

```diff
@@ -58,14 +56,13 @@
     @Test
     @WithMockUser
     public void shouldListToolGroups() {
-        final MultiValueMap<String, String> params = new LinkedMultiValueMap<>();
-        params.add(IDENTIFIER, TEST_STRING);
-        params.add(ACL_CLASS, ACL_CLASS_AS_STRING);
+        final MultiValueMap<String, String> params = multiValueMapOf(
```

Comment:
What do you think about inlining such maps creation and also extract each key-value pair on a separate lines:
```
final MvcResult mvcResult = performRequest(get(ENTITIES_URL)
                                           .params(multiValueMapOf(IDENTIFIER, TEST_STRING,
                                                                   ACL_CLASS, AclClass.DATA_STORAGE));
```